pipelines/powershell-package/templates/register.yaml

```diff
@@ -0,0 +1,7 @@
+parameters:
```

Comment:
missing # About this template comment
Why is this step here? Why we want to import all existing powershell modules in the repo?

I would also suggest changing the template name from register to "import all modules in this repo" or something similar

Reply:
This template is mean to just check if the module can be imported. I add some explanation into *About this template* header.